change-address/REPWRITERSPECS/BANNO.TRACKING8.FM

```diff
@@ -1,34 +1,36 @@
 [ File Name: BANNO.TRACKING8.FM
   
   Description: Batch PowerOn creates an FM report to update, 
-               expire or delete tracking type 8 records depending
-               on user selection.
+               expire or delete account level tracking type 8 
+               records depending on user selection.
                 
   
   Original Author: JuCarson 10/4/2022   R 2022.00
 
   Details:  
    Prompts:
     Select Run Mode:
-      Expire All Tracking 8s       - PowerOn will expire all unexpired tracking
-                                     type 8 records found on any open account.
+      Expire All Tracking 8s       - PowerOn will expire all unexpired account
+                                     level tracking type 8 records found on any
+                                     open account.
 
-      Delete All Tracking 8s       - PowerOn will delete all tracking type 8
-                                     records found on any open account.
+      Delete All Tracking 8s       - PowerOn will delete all account level tracking
+                                     type 8 records found on any open account.
 
       Delete Banno-specific Fields - PowerOn will delete any data found in
                                      USERCHAR12-USERCHAR15 of any unexpired
-                                     tracking type 8 records found on any 
-                                     open account.
+                                     account level tracking type 8 records found 
+                                     on any open account.
 
    Selection:
-    Any tracking type 8 record found on an open account.
+    Any account level tracking type 8 record found on an open account.
         
    Sorting:
     N/A
     
    Program Flow:
-    Program selects all open accounts with a tracking type 8 record. 
+    Program selects all open accounts with an account level tracking
+    type 8 record. 
 
         - PowerOn prints FM according to user input described above. 
 
```

Comment:
The code changes Look good Justin!

1. However; Please add some verbiage to the CFG Letter File, that mentions deleting the database check PowerOn and running this PowerOn if the FI has previously used address change and they are changing to prime name matching

2. I also suggest to be consistent, in the code read section,
I recommend changing "(0) Expire All Tracking 8s"
and "(1) Delete All Tracking 8s" to say All Account Level Tracking 8's

Reply:
Both completed